scavenger-agent-java/src/integrationTest/java/integrationTest/support/AgentIntegrationTestContextProvider.java

```diff
@@ -0,0 +1,72 @@
+package integrationTest.support;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.extension.Extension;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.extension.ParameterContext;
+import org.junit.jupiter.api.extension.ParameterResolutionException;
+import org.junit.jupiter.api.extension.ParameterResolver;
+import org.junit.jupiter.api.extension.TestTemplateInvocationContext;
+import org.junit.jupiter.api.extension.TestTemplateInvocationContextProvider;
+
+public class AgentIntegrationTestContextProvider implements TestTemplateInvocationContextProvider {
+    private static final String scavengerAgentPath = System.getProperty("integrationTest.scavengerAgent");
+    private static final String classpath = System.getProperty("integrationTest.classpath");
+    private static final String javaPaths = System.getProperty("integrationTest.javaPaths");
+
+    @Override
+    public boolean supportsTestTemplate(ExtensionContext context) {
+        assertNotNull(scavengerAgentPath, "This test must be started from Gradle");
+        assertNotNull(classpath, "This test must be started from Gradle");
+        assertNotNull(javaPaths, "This test must be started from Gradle");
```

Comment:
use assertj plz.

Reply:
fixed in [08d3010](https://github.com/naver/scavenger/pull/71/commits/08d3010fbf4dcd732fc8c68e73fd18de3d4d9209).